wire-ios-data-model/Source/MLS/OneOnOne/OneOnOneResolver.swift

```diff
@@ -62,12 +65,29 @@ public final class OneOnOneResolver: OneOnOneResolverInterface {
 
     // MARK: - Methods
 
+    public func resolveAllOneOnOneConversations(in context: NSManagedObjectContext) async throws {
+        let users: [ZMUser] = try await context.perform {
+            let request = NSFetchRequest<ZMUser>(entityName: ZMUser.entityName())
+            request.predicate = NSPredicate(format: "oneOnOneConversation != nil")
+            return try context.fetch(request)
+        }
+
+        for user in users {
+            guard let userID = await context.perform({ user.qualifiedID }) else {
+                WireLogger.conversation.error("required to have a user's qualifiedID to resolve 1-1 conversation!")
+                throw OneOnOneResolverError.otherUserNotFound
```

Comment:
do we stop the full process if we can't find one userID? should we continue instead to other users?

Reply:
Good catch, I have changed it